lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -270,52 +351,72 @@ $rich-list-item-icon-colors: (
 	}
 
 	&.-small {
-		min-height: 44px;
-		padding: $space-xxxxs $space-xxxs $space-xxxxs $space-xxxxs;
+		padding: $space-xxxxs;
 
 		@media #{breakpoint-s()} {
 			padding: 0 $space-xxxxs 0 0;
 		}
 
 		#{$root}__dragAndDrop,
 		#{$root}__iconWrapper {
-			padding: 0 0 0 $space-xs;
+			padding: $space-xxs $space-xxxxs 0 $space-xxs;
 
 			@media #{breakpoint-s()} {
 				padding: $space-xxxs 0 $space-xxxs $space-xs;
 			}
 		}
 
+		#{$root}__checkbox {
+			gap: $space-s;
+			padding: $space-xxxxxs $space-xxs 0 $space-xxs;
+
+			@media #{breakpoint-s()} {
+				gap: $space-xs;
+				padding: 0 $space-xs 0 $space-xxxxs;
+			}
+		}
+
 		#{$root}__content {
-			padding: 0 $space-xxxs 0 $space-xxs;
+			align-items: center;
+			padding: 0 $space-xxxxs 0 $space-xxs;
 		}
 
-		#{$root}__metaData {
```

Comment:
padding in size small, mobile layout for metaData should be 8px left, 4px right. Right now is 12px left and 4px right